src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -81,6 +81,8 @@ public class BuildApk : Task
 
 		public bool EnableLLVM { get; set; }
 
+		public bool AndroidEnableSGenConcurrent { get; set; }
```

Comment:
This property should be `EnableSGenConcurrent`, *or* you need to update the `.targets` file to set the `AndroidEnableSGenConcurrent` property.

The `BuildApk` properties need to be consistent with the `<BuildApk/>` task usage.